config_test_imdb.yml

```diff
@@ -0,0 +1,21 @@
+experiment_name: test_imdb
```

Comment:
Is it possible to remove this file and to add the dataset to `config.yaml` and `config_quick_debug.yml` file, in `scenario_params_list` / `dataset_name` please?

Reply:
Done, I had to create a new scenario: there is only 2 labels in the imdb dataset, so we can't create more than 2 clusters